src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -169,6 +177,35 @@ func useAllCores() {
 	runtime.GOMAXPROCS(numcores)
 }
 
+func configureLogging() *rotator.SizeRotator {
+	logdir := appdir.Logs("Lantern")
+	log.Debugf("Placing logs in %v", logdir)
+	if _, err := os.Stat(logdir); err != nil {
+		if os.IsNotExist(err) {
```

Comment:
I don't know how, but should also check if directory is writable. Just create a file to test?
> **drwxr-xr-x**  5 root  staff  170 Jan 15 19:47 /Users/merlin/Library/Logs/Lantern/